### PR TITLE
Progress Report - September 2022

### PR DESCRIPTION
**Pull request by goldenx86**
asdf

### CODE REVIEW

File: site/content/entry/yuzu-progress-report-sep-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+	"./ldn.png| The new Multiplyaer submenu"
+  >}}
+
+From there, double clicking a room will allow the user to join them. Keep in mind the Preferred Game is just a suggestion, there is no restriction on which games can be played in a room, unless an user hosted one wants to enforce it.
+
+{{< imgs
+	"./rooms.png| A lock means it's a password protected room"
+  >}}
+
+Speaking of user hosted rooms, the *Create Room* option in the Multiplayer menu allows users to host their own rooms, with custom player limits and password ecurity.
```
Comment (CeruleanSky) [inline site/content/entry/yuzu-progress-report-sep-2022/index.md:48]:
```suggestion
Speaking of user hosted rooms, the *Create Room* option in the Multiplayer menu allows users to host their own rooms, with custom player limits and password security.
```

File: site/content/entry/yuzu-progress-report-sep-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+After the Mainline release, Linux users reported to us that LDN rooms refused to work on some Linux distributions.
+The issue is in how the SSL package is distributed, so  [toastUnlimited](https://github.com/lat9nq), taking reference from a previous pull request from [Docteh](https://github.com/Docteh) {{< gh-hovercard "8933" " fixed the issue" >}} so now all supported operating systems can enjoy LDN.
+
+Some users reported crashes when playing `Pokémon Sword/Shield`. 
+While we implement the correction for it, make sure that a valid network device is selected in `Emulation > Configure… > System > Network > Network Interface`.
+The crash seems to happen when “None” is set as the network interface.
+
+## Graphic changes
+
+Nvidia OpenGL users OpenGL may have seen some strange graphical glitches in games like `Dragon Quest Builder`, `SNACK WORLD: THE DUNGEON CRAWL`, and `Pixel Game Maker Series Werewolf Princess Kaguya` due to an oversight in yuzu's handling of the performance-enhancing `vertex_buffer_unified_memory` extension.
```
Comment (CeruleanSky) [inline site/content/entry/yuzu-progress-report-sep-2022/index.md:95]:
```suggestion
Nvidia OpenGL users may have seen some strange graphical glitches in games like `Dragon Quest Builder`, `SNACK WORLD: THE DUNGEON CRAWL`, and `Pixel Game Maker Series Werewolf Princess Kaguya` due to an oversight in yuzu's handling of the performance-enhancing `vertex_buffer_unified_memory` extension. 
```

File: site/content/entry/yuzu-progress-report-sep-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+{{< single-title-imgs
+    "While this change helps Windows, Linux users will benefit the most of it"
+    "./scrollbug.png"
+    "./scrollfix.png"
+    >}}
+
+Docteh also fixed the {{< gh-hovercard "8909" "broken help page hyperlink on the `TAS` configuration window" >}}.
+
+{{< imgs
+	"./tas.png| Poeple seem to forget that blue underlined text is an hyperlink intended to be clicked"
```
Comment (CeruleanSky) [inline site/content/entry/yuzu-progress-report-sep-2022/index.md:248]:
```suggestion
	"./tas.png| People seem to forget that blue underlined text is an hyperlink intended to be clicked"
```

File: site/content/entry/yuzu-progress-report-sep-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+And Linux, of course. It's faster than Windows.
+
+## Hardware section
+
+Users started reporting crashes when booting up `Xenoblade Chronicles 3` on AMD Radeon GPUs running official AMD drivers (either the Windows driver, or the Linux amdvlk driver).
+We’re investigating and already passed AMD the relevant information.
+
+Affected users can try running [Mainline 1188](https://github.com/yuzu-emu/yuzu-mainline/releases/tag/mainline-0-1188) (doesn’t seem to be a consistent fix), or just run the game using OpenGL.
+The Mesa RADV driver is unaffected.
+
```
Comment (CeruleanSky) [inline site/content/entry/yuzu-progress-report-sep-2022/index.md:276]: Maybe a note saying  {{< gh-hovercard "8849" "astc: Enable parallel CPU astc decoding " >}} is merged now as this happened 3 days after the previous report was published and Maxwell and Pascal GPUs can upgrade their drivers, or if not, what the current status is for those GPUs and Vulkan.

Comment (goldenx86) [reply site/content/entry/yuzu-progress-report-sep-2022/index.md:281]: Thanks, added.

File: site/content/entry/yuzu-progress-report-sep-2022/index.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+For once, we don’t have much to say in this section.
+Or at least, we don’t want to start generating hype about new projects just yet.
+
+[Blinkhawk](https://github.com/FernandoS27) has started working on part 2 of `Project Y.F.C.`, as previously mentioned, and the rest of the GPU gang (Byte[], Maide, vonchenplus) is also busy with some top secret changes (for now, your writer has a reputation to uphold) to improve both rendering accuracy and performance.
+
+Having the first part of Y.F.C. merged to Mainline opens the floodgates to many projects that were on hold until now, we will talk about more of them in the next progress report.
+
+That’s all folks! As always, many thanks for your time. We hope to see you next.
+
+Special thanks to Mysterious Writer B for their help, and to CaptV0rt3x who came to aid at a moment’s notice.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-sep-2022/index.md:319]:
```suggestion
[Blinkhawk](https://github.com/FernandoS27) has started working on Part 2 of `Project Y.F.C.`, as previously mentioned, and the rest of the GPU gang (Byte[], Maide, vonchenplus) is also busy with some top secret changes (for now, your writer has a reputation to uphold) to improve both rendering accuracy and performance.

Having the first part of `Y.F.C.` merged to Mainline opens the floodgates to many projects that were on hold until now. We will discuss these in more detail in the next progress report.

Special thanks to Mysterious Writer B for their help with this progress report, and to CaptV0rt3x who came to aid at a moment’s notice.

That’s all folks! As always, many thanks for your time. We hope to see you in the next article!
```

**Review by Schplee — CHANGES_REQUESTED** [review]

File: site/content/entry/yuzu-progress-report-sep-2022/index.md [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+The Mesa RADV driver is unaffected.
+
+#### NVIDIA, one crash reason down, more remain
+
+We have merged the [multithreaded ASTC CPU decoder](https://github.com/yuzu-emu/yuzu/pull/8849) mentioned last month, so we strongly recommend NVIDIA Maxwell and Pascal (GTX 745/750, 900 and 1000 series) users to disable ASTC decoding if you run the latest drivers.
+
+You can find the option in `Emulation > Configure... > Graphics > Accelerate ASTC Texture Decoding`, an enabled setting uses the GPU to decode, while a disabled one uses the CPU instead.
+As a bonus, now this option can be a performance boost on systems with very weak integrated GPUs, like old laptops with MX series Geforce hardware, or any Intel iGPU, where the CPU would do a faster job dealing with ASTC decoding. Testing has shown that MX 500 and > Vega 3 tier hardware are slightly faster using the GPU for decoding.
+On affected NVIDIA hardware, the performance decrease while decoding ASTC textures is minimal. Avoiding a crash for example when opening the map in `The Legend of Zelda` games is arguably far more important.
```
[the remark this answers is not in the document]
Comment (goldenx86) [reply site/content/entry/yuzu-progress-report-sep-2022/index.md:285]: I mean bigger than 3.
Vega 3 is the model with 3 compute units, very slow. The next models, Vega 6 and higher are strong enough.

**Review by Schplee — CHANGES_REQUESTED** [review]

**Pull request merged**